doc/api/cluster.markdown

```diff
@@ -357,18 +357,19 @@ This example will echo back all messages from the master:
       });
     }
 
-### worker.suicide
+### worker.exitedVoluntarily (formerly worker.suicide)
 
 * {Boolean}
 
 Set by calling `.kill()` or `.disconnect()`, until then it is `undefined`.
```

Comment:
I would document worker.suicide and worker.exitedVolunatarily seperately, and for suicide docs, say its same as exitedVoluntarily, and exists for backwards compatibility.